### PR TITLE
Issue 3 edit an entry

### PR DESCRIPTION
**Pull request by DavidKyong**
Screen record in slack

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -18,26 +18,44 @@ $form.addEventListener('submit', function (event) {
...
+  }
 
   $image.setAttribute('src', '/images/placeholder-image-square.jpg');
-  $form.reset();
 
   viewSwap('entries');
   toggleNoEntries();
+  $form.reset();
+
+  location.reload();
```
Comment (shawnkost) [inline js/main.js]: You should not be reloading the page. We use `event.preventDefault` to specifically stop the page from refreshing. You should be finding the current `li` that exists on the DOM, and replacing it with the newly edited entry.

**Review by shawnkost — CHANGES_REQUESTED** [review]
One change here David! Let me know if you have any questions

File: js/main.js (hunk 1 above)
Comment (DavidKyong) [reply js/main.js]: I think I fixed the problem, but when I do replace the old form with the updated form, how come I have to set the data-entry-id back to the same entry-id I am fixing?

File: js/main.js [hunk 2]
```diff
@@ -30,11 +31,12 @@ $form.addEventListener('submit', function (event) {
...
   } else if (data.editing !== null) {
     const updateEntryId = data.editing.entryId;
-    const newData = data.entries.find(entry => entry.entryId === updateEntryId);
+    const $oldEntry = $newEntry.querySelector('[data-entry-id="' + updateEntryId + '"]');
 
-    newData.title = $title;
-    newData.notes = $message;
-    newData.url = $url;
+    const updateValue = renderEntry(formValue);
+    updateValue.setAttribute('data-entry-id', updateEntryId);
```
Comment (shawnkost) [inline js/main.js]: This should be happening inside of  `renderEntry`

**Review by shawnkost — CHANGES_REQUESTED** [review]
One more change!

File: js/main.js [hunk 3]
```diff
@@ -2,7 +2,7 @@ const $inputURL = document.querySelector('#url');
 const $image = document.querySelector('img');
 const $form = document.querySelector('form');
 const $ul = document.querySelector('ul');
-let updateEntryId;
+// let updateEntryId;
```
Comment (shawnkost) [inline js/main.js]: Delete all old code!

**Review by shawnkost — CHANGES_REQUESTED** [review]
One small edit here!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**